packages/core/admin/admin/src/components/MainNav/tests/NavBrand.test.tsx

```diff
@@ -0,0 +1,29 @@
+import { render, screen } from '@tests/utils';
+
+import { NavBrand } from '../NavBrand';
+
+jest.mock('../../../features/Configuration', () => ({
+  ...jest.requireActual('../../../features/Configuration'),
+  useConfiguration: () => ({
+    logos: {
+      menu: {
+        default: 'default',
+        custom: {
+          url: 'custom',
+        },
+      },
+    },
+  }),
+}));
```

Comment:
I think we have a `msw` endpoint for this, if you await one of the assertions it should work? 👀